.azure-pipelines/ipu-tests.yml

```diff
@@ -53,8 +53,8 @@ jobs:
         export GIT_TERMINAL_PROMPT=1
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'fairscale' not in line] ; open(fname, 'w').writelines(lines)"
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
-        python ./requirements/adjust_versions.py requirements/extra.txt
-        python ./requirements/adjust_versions.py requirements/examples.txt
+        python ./requirements/_adjust_versions.py requirements/extra.txt
+        python ./requirements/_adjust_versions.py requirements/examples.txt
```

Comment:
Can we keep the non-prefix name? This looks ugly and this code is not importable, we don't need to signify that this is "protected"

Reply:
ok, then lets use - instead of _ in the name to be sure it is not importable